test/resolve-path.spec.js

```diff
@@ -30,7 +30,7 @@ test('jspm import', assert => {
     .then(p => {
       // System.normalize in resolvePath will give us the absolute path
       const relative = '/' + path.relative(__dirname, url.parse(p).path);
-      assert.equal(relative, '/jspm_packages/npm/mock-package@1.0.0/mock-asset.scss', 'resolves "jspm:" import');
+      assert.equal(relative, '/' + path.join('jspm_packages', 'npm', 'mock-package@1.0.0', 'mock-asset.scss'), 'resolves "jspm:" import');
```

Comment:
Please use the shiny new ES2015 string concatenation syntax with backticks :wink: